<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>840</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Hollow</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.387, y1:0.407609, x2:0.39, y2:0.706087, stop:0 rgba(0, 230, 255, 255), stop:1 rgba(130, 255, 128, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>
                        :/pokemon/pokemonStart.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 56pt &quot;Pokemon Solid&quot;;
color:rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>POKIMON</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>610</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Pokemon Solid&quot;;
                        color: rgb(255, 255, 127);
                        background-color: rgb(0, 170, 255);
                        selection-background-color: rgb(255, 255, 127);
                        selection-color: rgb(255, 85, 0);
                        border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
  <include location="../Pokimon%20Master%20trainner/assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
